cmd/up.go

```diff
@@ -63,6 +66,7 @@ type UpCmd struct {
 
 	DotfilesSource string
 	DotfilesScript string
+	Token          string
```

Comment:
What do you think of putting this token into marimo specific settings? Similar to the `bindaddressoption`. 
This way we don't have to pollute all commands with it and can read the ide configuration on the agent through `workspaceInfo.IDE`

Reply:
100%

Reply:
actually I can't do that unless we are happy with the access token not changing? Currently we generate a new token every connection and exchange over a secure connection. Using the settings would require that the token not change :thinking:

Reply:
Is that a problem for any given session? 
You could stop it, change the token and restart the workspace 🤷‍♂️

Reply:
True, updated :)